# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/MediaManager.cpp
# Commit: 1248d2f659a9
# Full Hash: 1248d2f659a9f341f0de5d6558eecf364fc6f0a2
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2019-03-29 21:53:01
# Regressor Bug: 1494675
# File Overlap Count: 1
# Description:
#   Bug 1494675 - Remove windowID to MediaEngineSource mappings from MediaEngines. r=padenot
#   
#   These once served a caching-and-reuse purpose it seems, but it makes less sense
#   when sources are not shared.
#   
# ==============================================================================

diff -r 20543d03a4cd -r 1248d2f659a9 dom/media/MediaManager.cpp
--- a/dom/media/MediaManager.cpp	Wed Mar 27 14:10:24 2019 +0000
+++ b/dom/media/MediaManager.cpp	Wed Mar 27 14:10:10 2019 +0000
@@ -3435,13 +3435,6 @@
   MOZ_ASSERT(!GetWindowListener(aWindowID));
 
   RemoveMediaDevicesCallback(aWindowID);
-
-  MediaManager::PostTask(
-      NewTaskFrom([self = RefPtr<MediaManager>(this), aWindowID]() {
-        if (self->mBackend) {
-          self->mBackend->ReleaseResourcesForWindow(aWindowID);
-        }
-      }));
 }
 
 void MediaManager::RemoveMediaDevicesCallback(uint64_t aWindowID) {